sdk/communication/azure-communication-common/src/main/java/com/azure/android/communication/common/MicrosoftTeamsAppIdentifier.java

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.common;
+
+/**
+ * Communication identifier for a Microsoft Teams Application.
+ */
+public final class MicrosoftTeamsAppIdentifier extends CommunicationIdentifier {
+    private final String appId;
+    private final CommunicationCloudEnvironment cloudEnvironment;
+
+    /**
+     * Creates a MicrosoftTeamsAppIdentifier object
+     *
+     * @param appId The unique Microsoft Teams Application ID.
```

Comment:
```suggestion
     * @param teamsAppId The unique Microsoft Teams Application ID.
```
Let’s make sure to rename this in every instance where it appears.

Reply:
Discussed offline.